src/PlusVolumeReconstruction/vtkPlusPasteSliceIntoVolumeHelperCommon.h

```diff
@@ -112,6 +112,8 @@ struct vtkPlusPasteSliceIntoVolumeInsertSliceParams
   vtkImageData* outData;            // the output volume
   void* outPtr;                     // scalar pointer to the output volume over the output extent
   unsigned short* accPtr;           // scalar pointer to the accumulation buffer over the output extent
+  vtkImageData* imData;             // importance mask
```

Comment:
I know the other variables aren't very good, but would you mind renaming these to be more descriptive. If you're feeling generous, you could rename some others!